deploy/template.yaml

```diff
@@ -0,0 +1,179 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  ipv-cri-uk-passport-back
+
+Globals:
+  Function:
+    Timeout: 20
+
+Parameters:
+  Environment:
+    Type: String
+
+Resources:
+
+  IPVCriUKPassportAPI:
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: IPV CRI UK Passport API Gateway
+      StageName: !Sub ${Environment}
+
+  IPVDcsCredentialFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-dcs-credential-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.lambda.DcsCredentialHandler::handleRequest
+      Runtime: java11
+      PackageType: Zip
+      CodeUri: ../lambdas/dcscredential
+      Architectures:
+        - x86_64
+      MemorySize: 512
+      Environment:
+        Variables:
+          ENVIRONMENT: !Sub "${Environment}"
+          DCS_RESPONSE_TABLE_NAME: !Select [1, !Split ['/', !GetAtt DCSResponseTable.Arn]]
+          CRI_PASSPORT_ACCESS_TOKENS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CRIPassportAccessTokensTable.Arn]]
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref DCSResponseTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAccessTokensTable
+      Events:
+        IPVCriUKPassportAPI:
+          Type: Api
+          Properties:
+            RestApiId: !Ref IPVCriUKPassportAPI
+            Path: /credential
+            Method: GET
+
+  IPVDcsAccessTokenFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-dcs-token-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.accesstoken.AccessTokenHandler::handleRequest
+      Runtime: java11
+      PackageType: Zip
+      CodeUri: ../lambdas/accesstoken
+      Architectures:
+        - x86_64
+      MemorySize: 512
+      Environment:
+        Variables:
+          ENVIRONMENT: !Sub "${Environment}"
+          CRI_PASSPORT_AUTH_CODES_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CRIPassportAuthCodesTable.Arn]]
+          CRI_PASSPORT_ACCESS_TOKENS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CRIPassportAccessTokensTable.Arn]]
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAuthCodesTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CRIPassportAccessTokensTable
+      Events:
+        IPVCriUKPassportAPI:
+          Type: Api
+          Properties:
+            RestApiId: !Ref IPVCriUKPassportAPI
+            Path: /token
+            Method: POST
+
+  IPVPassportFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "ipv-passport-${Environment}"
+      Handler: uk.gov.di.ipv.cri.passport.passport.PassportHandler::handleRequest
```

Comment:
This one is correct but the passport.passport seems a bit weird, might mean we change that in the future maybe? But not necessarily something to worry about in this PR

Reply:
Yeah I also hate this 😂  but it follows the pattern, I'd like us to maybe rethink the lambda name?

Reply:
Yeah good shout, I think the lambda names in general can be better so something we can fix after